README.md

```diff
@@ -180,6 +180,34 @@ room {
 * There can only be a single schema export directory for the project - you cannot configure variant-specific export
 directories.  Schemas exported from different variants will be merged in the directory specified in the "room" extension.
 
+#### Ksp
+Since version 2.6.6, `RoomSchemaLocationWorkaround` supports Kotlin Symbol Processing(Ksp). Like kapt, applying the Room
```

Comment:
I think we should bump to 2.7 given this is a big change.

Reply:
done